pkg/workspace/data_values_pre_processing.go

```diff
@@ -114,6 +112,10 @@ func (pp DataValuesPreProcessing) typeAndCheck(dataValuesDoc *yamlmeta.Document)
 	if len(chk.Violations) > 0 {
 		return chk
 	}
+
+	// updates node's validations meta from Node's assigned type
+	yamlmeta.Walk(dataValuesDoc, schema.AssignSchemaValidations{})
```

Comment:
```suggestion
	_ = yamlmeta.Walk(dataValuesDoc, schema.AssignSchemaValidations{})
```

Let's ignore _explicitly_ to differentiate from cases where we forget to handle errors.